lib/Schedule/LongSteps/Storage/DBIxClass.pm

```diff
@@ -200,4 +206,44 @@ sub find_process{
     return $self->_get_resultset()->find({ id => $process_id });
 }
 
+=head2 update_process
+
+Overrides L<Schedule::LongSteps::Storage#update_process> to add
```

Comment:
FYI Moose provides an `override` statement to formally override a method

~~~perl
override update_process => sub { ... }
~~~

Reply:
done that! Ta